src/components/DefinitionList/types.ts

```diff
@@ -0,0 +1,37 @@
+import {QAProps} from '@gravity-ui/uikit';
+
+import type {HelpPopoverProps} from '../HelpPopover';
+export type DefinitionListItemNote = string | HelpPopoverProps;
+
+export interface DefinitionListGroupItem {
+    label: string;
+}
+
+export interface DefinitionListGroup {
+    label: string;
+    items?: DefinitionListItem[];
+}
+
+export interface DefinitionListItem {
+    name: React.ReactNode;
+    content?: React.ReactNode;
+    contentTitle?: string;
+    nameTitle?: string;
+    copyText?: string;
+    note?: DefinitionListItemNote;
+    multilineName?: boolean;
+}
+
+export interface DefinitionListItemGrouped extends DefinitionListItem {
+    isGrouped?: boolean;
+}
+
+export interface DefinitionListProps extends QAProps {
+    items: (DefinitionListItem | DefinitionListGroup)[];
```

Comment:
We should keep previous type signature: `items: DefinitionListItem[]`. So `DefinitionListItem` should be defined as follows:
```ts
type DefinitionListItem = DefinitionListSingleItem | DefinitionListGroupItem
```